src/content/en/fundamentals/media/manipulating/cheatsheet.md

```diff
@@ -191,11 +194,6 @@ remove all whitespace including the final carriage return.
 For the `-key` flag use the key created earlier and stored in the media.key
 file. However, when entering it on the command line, be sure you've removed its
 whitespace. For the `-key_id` flag repeat the key value.
```

Comment:
"It can usually be found in the manifests and media containers in plain text/hex." What manifests and media containers?